### ISSUE TITLE
Add options to merge if CI passes

### ISSUE THREAD

**Issue by Cogitri** [maxice8/mkmr#33]
I guess mgmr would need to stay alive in the background until CI passes with this if multiple MRs are specified (since the 2nd one would have to be rebased after the 1st one has been merged), so I'm not sure how practical this is.

**Comment by maxice8** [comment]
That would lead to mgmr basically becoming a daemon while waiting for CI to pass, if we tell MRFOO to pass only on CI then merge MRBAR in the meantime then we have to do MRFOO all over again.

The intended usecase is where you already know if it works, or CI Fails but you know it works.

**Comment by Cogitri** [comment]
Yeah, I guess  I just kind of want this to emulate a merge train, but that probably isn't a goal of mgmr.

**Comment by maxice8** [comment]
The way the workqueue is currently structured it means it is a merge train but without waiting for CI, if we wait for CI it means we will have to keep polling the API for a bit.
